src/Illuminate/Support/Str.php

```diff
@@ -149,9 +149,10 @@ public static function finish($value, $cap)
      *
      * @param  string|array  $pattern
      * @param  string  $value
+     * @param  string $wildcard
```

Comment:
The convention is two spaces:
```suggestion
     * @param  string  $wildcard
```